vendor/github.com/aws/aws-sdk-go/service/elbv2/api.go

```diff
@@ -7586,6 +7586,10 @@ type TargetGroupAttribute struct {
 	//    from draining to unused. The range is 0-3600 seconds. The default value
 	//    is 300 seconds.
 	//
+	//    * slow_start.duration_seconds - The amount time for targets to warm up
```

Comment:
This is failing CI due to:

```
$ make vendor-status
The following packages are missing or modified locally:
	github.com/aws/aws-sdk-go/service/elbv2
Error: status failed for 1 package(s)
make: *** [vendor-status] Error 2
The command "make vendor-status" exited with 2.
```

When it comes to vendor updates we usually:
* Keep them separate from other code updates
* Use `govendor` to update the dependencies (for now)

I'll go ahead and submit the v1.13.55 update PR right now then you can rebase this PR on that.

Reply:
Dependency PR submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/4663

Reply:
The full SDK update has been merged into master 👍